Cargo.toml

```diff
@@ -26,8 +26,9 @@ semver = "1"
 serde = { version = "1", features = ["derive"] }
 serde_json = { version = "1", features = ["preserve_order", "std"] }
 thiserror = "1"
-typed-builder = "0.14"
+typed-builder = "0.15"
 validator = { version = "0.16", features = ["derive"], optional = true }
+proc-macro2 = "1.0.66"
```

Comment:
What does this do?

Reply:
That's leftover of me trying to fix the CI checks. I'll tidy things up shortly.